puppetlabs_spec_helper.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'puppet-lint', '~> 4.0'
   spec.add_runtime_dependency 'puppet-syntax', '~> 4.1', '>= 4.1.1'
   spec.add_runtime_dependency 'rspec-github', '~> 2.0'
-  spec.add_runtime_dependency 'rspec-puppet', '~> 4.0'
+  spec.add_runtime_dependency 'rspec-puppet', '>= 4.0', '< 6'
```

Comment:
for Vox Pupuli we would probably use `~> 5` to ensure we use the latest version. I don't know what's the preference for Perforce.